src/3k-mapper.xml

```diff
@@ -438,8 +443,12 @@ end</script>
                 <Alias isActive="yes" isFolder="no">
                     <name>Show Exits</name>
                     <script>--show room exits
-tmap:echo(getRoomExits(tmap.lastId))
-tmap:echo(&quot;special exits:\n&quot;)
+local exits = getRoomExits(tmap.lastId)
```

Comment:
`getRoomExits( roomId )` returns nothing if the room does not exist - so that means exits will be `nil` in that case. Do you need to have a check for that?